earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TMPFILE=/tmp/FusionTutorial-Full.tar.gz
+SCRIPTDIR=`dirname $0`
+
+if [ ! -d Imagery ]; then
+	wget http://data.opengee.org/FusionTutorial-Full.tar.gz -O $TMPFILE
+	tar -xvzf $TMPFILE -C $SCRIPTDIR
```

Comment:
I'd, personally, quote all variables that need to expand to one command argument, even for `wget` and `rm`, although the `$TMPFILE` virable doesn't contain any special characters.